### ISSUE TITLE
No Dropdowns Visible in MST production Instance

### ISSUE THREAD

**Issue by mryanlord** [MDMI/mdmiecore#152]
Logged in as multiple users in Dev Mode Test Tools: RiversideOrg@gmail.com and MSTSuperuser@gmail.com. Both users could not see any map choices in drop down menu (though there was a small change denoting active use of dropdown). 

Using Google Chrome 67.0.3396.99 and Firefox 61.0.1 on Windows 10 Pro.

**Comment by swmuir** [comment]
Which url ?

**Comment by swmuir** [comment]
Fixed - need to use domain name to see the transform

**Comment by swmuir** [comment]
Fixed the web app was not able to connect to the sevice using ip after the update in the ssl

need to use the www.mdixnet domain

**Comment by mryanlord** [comment]
I used this URL provided in an email thread: 
https://www.mdixinc.net:8443/mdixui/ 

I just tried there and got the same issue. I'm working on clearing cache etc for this site and try again

**Comment by swmuir** [comment]
yep thats the right one - i fixed the issue it was related to the ssl on the site

should be working now

**Comment by mryanlord** [comment]
Got it to work in Firefox
